static/js/components/EducationTab.js

```diff
@@ -37,6 +40,15 @@ class EducationTab extends ProfileTab {
     this.handleCloseDialog = this.handleCloseDialog.bind(this);
 
   }
+
+  componentDidMount() {
+    // Use dialog polyfill to allow dialog usage on browsers other than Chrome and Opera (as of early 2016)
+    const node = ReactDOM.findDOMNode(this);
+    for (let dialog of node.querySelectorAll("dialog")) {
+      dialogPolyfill.registerDialog(dialog);
+    }
+  }
```

Comment:
We can only do this once, otherwise we get a warning. I don't know if the polyfill has a way of unregistering the nodes